framework/src/modules/chain/modules/rounds.js

```diff
@@ -247,12 +379,20 @@ Rounds.prototype.tick = function(block, done, tx) {
 			});
 	}
 
+	// Start round ticking
+	__private.ticking = true;
 	async.series(
 		[
+			function updateRoundInformation(cb) {
+				__private.updateRoundInformationForTransactions(
+					round,
+					block.transactions,
+					true,
```

Comment:
In recent code, we are using a constant approach rather than passing static values. e.g. 

```
const isForwardTick = true; 
```

And then pass `isForwardTick` to function, that means code more readable.